shared/js/dataDriver.js

```diff
@@ -85,6 +85,18 @@ loop.DataDriver = function() {
     }
 
     /**
+      * Add a page to the room's table of contents.
+      *
+      * @param {String} url       The url of the page.
+      * @param {String} title     The title of the page.
+      * @param {Object} metadata  Additional metadata,
```

Comment:
Any reason to have these as individual parameters instead of a single page object?